### DISCUSSION TITLE
About dashboard manipulation

### DISCUSSION THREAD

**Discussion by Kushaldotel** [farridav/django-jazzmin#493] [Q&A]
![Screenshot 2023-05-04 164307](https://user-images.githubusercontent.com/114340617/236419972-a4b0ba8d-a9d8-4ba3-8df1-b9628259625d.png)
I am not able to insert any thing in my dashboard section. Please see this image. In the dashboard section I want to insert Images like Barcharts, Piecharts and other things. But i am able to see only the 'Authentication and Authorization', my django app name and Model and Recent actions. I want to overwrite them. I just wanna customize the dashboard section. The url is localhost/admin/. in the same url i want the things. In in the right of side bar i want the images in dashboard section for fetching the real datas.
![Screenshot 2023-05-05 150223](https://user-images.githubusercontent.com/114340617/236420753-9181f2f1-a5f2-4796-8ac4-6abfd9e82b82.png)
Like this.

**Comment by sadiqaldawas** [comment]
It very important to do that like filament.
